packages/Search/src/DTK_LinearBVH_decl.hpp

```diff
@@ -23,6 +23,12 @@
 
 namespace DataTransferKit
 {
+namespace Details
+{
```

Comment:
Then let's pick another name. If I see details, I will look into the directory `details`.